profiles/management/commands/gen_realistic_search_data.py

```diff
@@ -248,10 +275,13 @@ def handle(self, *args, **options):
         if len(user_data_list) == existing_fake_user_count and len(program_data_list) == existing_fake_program_count:
             self.stdout.write("Realistic users and programs appear to exist already.")
         else:
-            new_program_count = deserialize_program_data_list(program_data_list)
+            new_programs = deserialize_program_data_list(program_data_list)
+            if options.get('staff_user', None):
```

Comment:
I think the default 2nd argument for `.get` is already `None`, is there a python subtlety I'm missing?